packages/sdk/src/transactions/types.ts

```diff
@@ -0,0 +1,166 @@
+import type { Inscription, Ordinal } from "../inscription/types";
+
+// Types are based on bitcoin RPC
+// Refer to https://bitcoincore.org/en/doc/25.0.0/rpc/rawtransactions/decoderawtransaction/
+
+/**
+ * Transaction output (Vout)
+ */
+export type Vout = {
+  /**
+   * Value in BTC
+   */
+  value: number;
+
+  /**
+   * Index
+   */
+  n: number;
+
+  /**
+   * Ordinals
+   */
+  ordinals: Ordinal[];
+
+  /**
+   * Inscriptions
+   */
+  inscriptions: Inscription[];
+
+  /**
+   * Is spent
+   */
+  spent: string | false;
```

Comment:
@iamcrazycoder is `spent` a standard field? couldn't find it in the bitcoin RPC.

Reply:
Its an internal prop that we added to track the status of an UTXO. A few months ago, we introduced a virtual mempool and started keeping track of UTXOs that relayed through our nodes.